packages/api/app/routes.js

```diff
@@ -103,7 +103,7 @@ router.use('/ems', ensureAuthorized, ems);
 // eslint-disable-next-line no-unused-vars
 router.use((error, req, res, next) => {
   log.error(error);
-  return res.status(500).send({ error: error.message });
+  return res.boom.badRequest(error.message, error);
```

Comment:
Using `badImplementation` for some reason with `boom` obscures your error message

Reply:
BadImplementation is a 5xx error and those are hidden from the user. 
https://hapi.dev/family/boom/?v=8.0.1#boombadimplementationmessage-data---alias-internal
"All 500 errors hide your message from the end user. Your message is recorded in the server log."

Reply:
Yep, which is why I used `badRequest`, a 400, instead.

Reply:
ok, I just saw "for some reason" and thought I was helping. :)